Dockerfile

```diff
@@ -1,18 +1,14 @@
-FROM ubuntu:18.04
-MAINTAINER @aruizca - Angel Ruiz
+FROM anapsix/alpine-java
```

Comment:
Alpine is not currently supported by Confluence: https://confluence.atlassian.com/doc/supported-platforms-207488198.html

Reply:
Fixed!